components/Avatar/avatar.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Fontawesome Icons -->
+    <script src="https://kit.fontawesome.com/dd0d05bba7.js" crossorigin="anonymous"></script>
+
+    <!-- My stylesheets -->
+
+    <link rel="stylesheet" href="/css/main.css">
+    <link rel="stylesheet" href="/components/Avatar/avatar.css">
+    <title>Pastel UI | Avatar</title>
+</head>
+
+<body>
+    <div class="wrapper">
+        <div class="heading">
+            <h1>Avatar</h1>
+            <p>The Avatar component is used to represent a user, and displays the profile picture, initials or fallback
+                icon.</p>
+        </div>
+
+        <hr class="divider">
+
+        <div class="sub-heading">
+            <h3>Standard Avatar</h3>
+            <p>Standard Avatar comes in a single size.</p>
+            <p>You can use the <strong>avatar</strong> class on your figure tag to get the standard avatar.</p>
+        </div>
+
+        <div class="avatars-wrapper">
+            <figure class="avatar">
+                <img src="/images/avatar-img-1.jpg" alt="avatar-image-1" class="avatar-img" />
+
+            </figure>
+            <figure class="avatar">
+                <img src="/images/avatar-img-2.jpg" alt="avatar-image-2" class="avatar-img" />
+
+            </figure>
```

Comment:
remove these spaces use prettier or any other code formating tool

Reply:
Done. Refactored the code and used Prettier